x11-misc/xbattbar/xbattbar-1.4.9.ebuild

```diff
@@ -33,17 +32,15 @@ src_prepare() {
 	sed -i \
 		-e "s:usr/lib:usr/$(get_libdir):" \
 		xbattbar.c || die
+}
 
-	tc-export PKG_CONFIG
+src_configure() {
+	tc-export CC PKG_CONFIG
 }
 
 src_compile() {
 	use kernel_linux && append-flags -Dlinux
```

Comment:
I’d love this to configure and then drop the complete phase

Reply:
(It might be worth a grep for stuff like -Dlinux in the tree and see how much of it is unguarded by the way...)